src/app/components/dropdown/index.html

```diff
@@ -23,6 +23,12 @@
       defaultValue="default">
       Specifies a background color for the dropdown button. If unspecified, the <stache-code>default</stache-code> value sets the background color to <stache-code>sky-color-white</stache-code>. To change the background color to <stache-code>sky-color-blue</stache-code>, specify <stache-code>primary</stache-code>.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissOnBlur"
+      isOptional="true"
+      defaultValue="true">
+      Specifies a Boolean value to indicate whether the dropdown closes when a user clicks away from the menu.
```

Comment:
For Boolean values, we usually start with "Indicates whether to..." and go from there. So I'd delete "Specifies a Boolean value to," and I'd flip "the dropdown closes" around to "to close the dropdown." 

We also try to stick with plural ("users") vs. singular ("user") whenever possible (i.e., unless plural becomes awkward or confusing), so I'd change "a user clicks" to "users click."

Also, returning to "Specifies a Boolean value to," we don't generally call out the type, but I talked with Steve just today about whether it's useful to include type information in property descriptions because I was updating a topic that does call out that information. Based on that conversation, I'm planning to start including type information in the docs, but it will be in a separate sentence at the end of the property description. So to be consistent with this new style, I'd add "This property accepts <stache-code>boolean</stache-code> values." after the first sentence.